themes/radial/components/gantt-chart.typ

```diff
@@ -0,0 +1,135 @@
+#import "../colors.typ": *
+#import "../icons/icons.typ": *
+#import "@preview/timeliney:0.0.1"
+
+/// Creates a gantt chart for task management.
+///
+/// Example Usage:
+///
+/// ```typ
```

Comment:
Frankly I'd really like to see this API improved. The fact that a single call with only 4 tasks takes up almost as many lines (1 less) than the actual implementation should be a massive red flag to you.

I decent simplification could be to provide `start_date` and `end_date` parameters, instead of having users manually specify the amount of weeks and months, along with their names.